packages/components/src/components/primitives/Icons/types.ts

```diff
@@ -1,4 +1,7 @@
+const iconSizes = ['s', 'm', 'l'] as const;
+type IconSizes = typeof iconSizes[number];
```

Comment:
We might want to discuss plural vs singular because right now we have different approaches in other components @tkanzakic. For example `Button` uses `ButtonSizes` const and `ButtonSize` type. (Also notice the uppercase). 

Maybe we can correct this in a different PR trying to make all of the types consistent across the different components.

Reply:
the type should be singular